src/utils/index.js

```diff
@@ -0,0 +1,15 @@
+import childrenValueInputValidation from './childrenValueInputValidation';
+import createChainedFunction from './createChainedFunction';
+import createContextWrapper from './createContextWrapper';
+import CustomPropTypes from './CustomPropTypes';
+import domUtils from './domUtils';
+import ValidComponentChildren from './ValidComponentChildren';
+
+export default {
+  childrenValueInputValidation,
+  createChainedFunction,
+  createContextWrapper,
```

Comment:
I'd really rather this not be exposed. There's intentionally no user-facing documentation for any internal users of this function.